docs/user-guide/deferred-download.rst

```diff
@@ -26,21 +26,33 @@ with the Pulp core services.
 
 Deferred downloading relies on three services:
 
-* ``httpd`` - The Apache web server acts as a TLS termination point and reverse proxy.
-  This service handles the incoming client requests and forwards them to Squid.
+* A reverse proxy that terminates the TLS connection. In this guide Apache is configured
```

Comment:
You might s/Apache/httpd/g or s/Apache/Apache httpd/g or s/Apache/Apache HTTP Server/g throughout this page. I think most people do know what "Apache" means, but it could be good to use the real name. It's a nitpick, feel free to ignore.